CMakeLists.txt

```diff
@@ -162,7 +159,21 @@ set(NRN_PROFILER
 # =============================================================================
 # Set Python additional versions earlier (especially for old CMake)
 # =============================================================================
-set(Python_ADDITIONAL_VERSIONS 3 3.9 3.8 3.7 3.6 3.5 3.4 3.3 3.2 3.1 3.0 2.8 2.7 2.6)
+set(Python_ADDITIONAL_VERSIONS
+    3
+    3.9
+    3.8
+    3.7
+    3.6
+    3.5
+    3.4
+    3.3
+    3.2
+    3.1
+    3.0
+    2.8
+    2.7
+    2.6)
```

Comment:
`just nitpick` : do you want to fence it or just keep it like this?

Reply:
I don't mind. I'll leave it alone unless someone has a stronger view. `FindPythonInterp` and `FindPythonLibs`, which apparently use this variable, are deprecated in CMake 3.12+...https://github.com/neuronsimulator/nrn/pull/1695, https://github.com/neuronsimulator/nrn/issues/1700.

Reply:
Ok. Lets cleanup FindPythonInterp part separately.